examples/elixir/get_started/echoer.exs

```diff
@@ -2,12 +2,12 @@ defmodule Echoer do
   use Ockam.Worker
 
   alias Ockam.Message
-  alias Ockam.Router
+  alias Ockam.Worker
 
   @impl true
   def handle_message(message, %{address: address} = state) do
     IO.puts("Address: #{address}\t Received: #{inspect(message)}")
-    Router.route(Message.reply(message, address, Message.payload(message)))
+    Worker.route(Message.reply(message, address, Message.payload(message)), state)
```

Comment:
This is a good example of where the change make sense.  This _is_ a worker,  and you have the worker state to pass,   `Worker.route(message, state)` make sense.   And that right now just discard the state parameter and call Router.route/1,  that's ok and what the issue was about 👍